src/library_html5.js

```diff
@@ -1911,14 +2191,23 @@ var LibraryJSEvents = {
 
   emscripten_set_webglcontextlost_callback: function(target, userData, useCapture, callbackfunc) {
     JSEvents.registerWebGlEventCallback(target, userData, useCapture, callbackfunc, {{{ cDefine('EMSCRIPTEN_EVENT_WEBGLCONTEXTLOST') }}}, "webglcontextlost");
```

Comment:
note to self: seems emscripten_set_webglcontextlost_callback: should be deleted?